docker/Dockerfile

```diff
@@ -38,6 +38,12 @@ RUN --mount=type=cache,target=/lemmy/target set -ex; \
         cargo clean --release; \
         cargo build --features "${CARGO_BUILD_FEATURES}" --release; \
         mv target/"${RUST_RELEASE_MODE}"/lemmy_server ./lemmy_server; \
+        #
+        # Compress the binary with upx
+        wget https://github.com/upx/upx/releases/download/v4.2.4/upx-4.2.4-amd64_linux.tar.xz; \
+        tar -xvf upx-4.2.4-amd64_linux.tar.xz; \
+        cp upx-4.2.4-amd64_linux/upx /usr/bin; \
+        upx --best --lzma lemmy_server; \
```

Comment:
Unfortunately upx / upx-upl isn't in the debian bookworm repositories 🤷 . Only previous debian releases.